### PR TITLE
Fix yul example in docs

### PR DESCRIPTION
**Pull request by vedant1811**
Fixes issue: https://github.com/ethereum/solidity/issues/6489#issuecomment-490602073

### CODE REVIEW

File: docs/yul.rst [hunk 1]
```diff
@@ -51,16 +51,16 @@ and ``mod`` are available either natively or as functions and computes exponenti
 .. code::
```
Comment (chriseth) [inline docs/yul.rst:52]: Please also add in the paragraph above something like ``As per the warning above, the following code is untyped and can be compiled using `solc --strict-assembly` ``.

File: docs/yul.rst [hunk 2]
```diff
@@ -593,15 +593,21 @@ An example Yul Object is shown below:
...
-            let offset = allocate(size)
+            let size := datasize("runtime.Contract2")
+            let offset := allocate(size)
             // This will turn into a memory->memory copy for eWASM and
             // a codecopy for EVM
             datacopy(offset, dataoffset("runtime.Contract2"), size)
             // constructor parameter is a single number 0x1234
             mstore(add(offset, size), 0x1234)
-            create(offset, add(size, 32))
+            create(offset, size, 32)
```
Comment (chriseth) [inline docs/yul.rst]: I think it should be
```suggestion
            create(offset, add(size, 32), 0)
```

File: docs/yul.rst [hunk 3]
```diff
@@ -617,16 +623,22 @@ An example Yul Object is shown below:
...
-                let offset = allocate(size)
+                let size := datasize("Contract2")
+                let offset := allocate(size)
                 // This will turn into a memory->memory copy for eWASM and
                 // a codecopy for EVM
                 datacopy(offset, dataoffset("Contract2"), size)
                 // constructor parameter is a single number 0x1234
                 mstore(add(offset, size), 0x1234)
-                create(offset, add(size, 32))
+                create(offset, size, 32)
```
Comment (chriseth) [inline docs/yul.rst]: Same here.

File: docs/yul.rst [hunk 4]
```diff
@@ -47,20 +47,21 @@ if the backend changes. For a list of mandatory built-in functions, see the sect
 
 The following example program assumes that the EVM opcodes ``mul``, ``div``
 and ``mod`` are available either natively or as functions and computes exponentiation.
+As per the warning above, the following code is untyped and can be compiled using `solc --strict-assembly`
```
Comment (chriseth) [inline docs/yul.rst]: Please use double backticks.

**Comment by chriseth** [comment]
Nice! Please squash into a single commit or at least squash the last commit.

**Review by leonardoalt — APPROVED** [review]

**Pull request merged**